includes/feeds/google.php

```diff
@@ -556,10 +557,16 @@ public function make_request($id = '', $time_min = 0, $time_max = 0)
 			}
 
 			$is_authhelper = get_option('simple_calendar_run_oauth_helper');
+			$feed_type = wp_get_object_terms($post_id, 'calendar_feed');
+
 			// Query events in calendar.
 			$simple_calendar_auth_site_token = get_option('simple_calendar_auth_site_token');
 			$response = '';
-			if (isset($simple_calendar_auth_site_token) && !empty($simple_calendar_auth_site_token && $is_authhelper)) {
+			if (
+				isset($simple_calendar_auth_site_token) &&
+				!empty($simple_calendar_auth_site_token && $is_authhelper) &&
```

Comment:
Is this condition correct `!empty($simple_calendar_auth_site_token && $is_authhelper)`? Why two conditions inside `empty`?